.github/workflows/test.yml

```
@@ -154,27 +114,26 @@ jobs:
+          poetry run isort .
 
       - name: Check remaining issues
         run: |
-          flake8 . || echo "⚠️ Flake8 issues found - please fix manually"
-          mypy backend/core --ignore-missing-imports || echo "⚠️ Type checking issues found - please fix manually"
-
-      - name: Security scan
-        run: |
-          safety check || echo "⚠️ Security vulnerabilities found - please review"
+          poetry run flake8 . || echo "⚠️ Flake8 issues found - please fix manually"
+          poetry run mypy backend/core --ignore-missing-imports || echo "⚠️ Type checking issues found - please fix manually"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)